main.py

```diff
@@ -34,7 +36,7 @@ def main():
         if not task.has_validation_docs():
             continue
         result = task.evaluate(
-            docs=task.validation_docs(),
+            docs=itertools.isslice(task.validation_docs(), stop=args.limit),
```

Comment:
```suggestion
            docs=itertools.islice(task.validation_docs(), args.limit),
```

Reply:
- isslice -> islice (typo)
- needs a start position of 0 ie. (iterable, start, stop[, step])
- updated named -> positional argument